Formula/ssdb.rb

```diff
@@ -46,37 +46,10 @@ def install
     etc.install "ssdb_slave.conf"
   end
 
-  plist_options manual: "ssdb-server #{HOMEBREW_PREFIX}/etc/ssdb.conf"
-
-  def plist
-    <<~EOS
-      <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-      <plist version="1.0">
-        <dict>
-          <key>KeepAlive</key>
-          <dict>
-            <key>SuccessfulExit</key>
-            <false/>
-          </dict>
-          <key>Label</key>
-          <string>#{plist_name}</string>
-          <key>ProgramArguments</key>
-          <array>
-            <string>#{opt_bin}/ssdb-server</string>
-            <string>#{etc}/ssdb.conf</string>
-          </array>
-          <key>RunAtLoad</key>
-          <true/>
-          <key>WorkingDirectory</key>
-          <string>#{var}</string>
-          <key>StandardErrorPath</key>
-          <string>#{var}/log/ssdb.log</string>
-          <key>StandardOutPath</key>
-          <string>#{var}/log/ssdb.log</string>
-        </dict>
-      </plist>
-    EOS
+  service do
+    run [opt_bin/"ssdb-server", etc/"ssdb.conf"]
+    working_dir var.to_s
+    error_log_path var/"log/ssdb.log"
```

Comment:
Doesn't this also need
```ruby
log_path var/"log/ssdb.log"
```
? Same for a few others; this corresponds to `StandardOutPath`.